src/Components/TimeTable/Timetrack/Timetrack.css

```diff
@@ -1,12 +1,16 @@
 .time-track {
-    width: 1590px;
+  width: 1590px;
+  position: sticky;
```

Comment:
can you give this a border-bottom 1px too? Looks weird when it loses the border. You might need to swap the border on the stages to bottom instead of top?